.pre-commit-config.yaml

```diff
@@ -24,8 +24,12 @@ repos:
       - id: mypy
         exclude: ^docs/conf.py
 
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.2.1
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.5
     hooks:
-      - id: prettier
-        args: [--prose-wrap=always, --print-width=88]
+      - id: mdformat
+        args: [--number, --wrap=88]
+        additional_dependencies:
+          - mdformat-gfm
+          - mdformat-myst
+        exclude: ^.github/ISSUE_TEMPLATE
```

Comment:
It should be safe to format these as long as `frontmatter` syntax support is enabled (which it is because `mdformat-myst` includes it).

You were worried about safety in the commit message. Note that mdformat has a check to ensure AST stays unchanged (similar to Black), which Prettier to my knowledge doesn't have.

Reply:
Alright you've convinced me :slightly_smiling_face: You should definitely advertise the AST safety feature of mdformat IMO (why not mention it in your "why not prettier" section).